blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -37,7 +37,7 @@ const ResultsList = ({ customFields }) => {
 		? deployment(`${contextPath}/${fallbackImage}`)
 		: fallbackImage;
 	const promoElements = resolveDefaultPromoElements(customFields);
-	const phrases = usePhrases();
+	const phrases = getTranslatedPhrases(locale || "en");
```

Comment:
You can undo this change - the `usePhrases` hook is a helper to reduce the need to look up all the necessary items ahead of time

Reply:
Reverted to usePhrases